### ISSUE TITLE
It it fine to use all different distribution model when scale up animal PBPK model to human PBPK model?

### ISSUE THREAD

**Issue by ImSuajo** [Open-Systems-Pharmacology/Suite#128]
Hello.
I'm trying to scale up animal PBPK model to human PBPK model.
I make mouse, rat, dog PBPK model but there are not correaltion between these species.
So I choose all diffrent distribution model to anmial PBPK model. (rat: Poulin and Theil, mouse: Berezhovsky, dog: Pk-sim standard)
Is it no problem I choose diffrent distribution model and optimize the drug parameter together?
And also I want to know If i try to scale up these model to human, which distribution model I have to choose fot human PBPK model?

**Comment by HenrikCordes** [comment]
Hi @ImSuajo,

the intention behind the different distribution models it to describe the passive distribution of a compound into different body tissues. In my opinion, the distribution model should be choose compound specific. Meaning, one should find/choose one distribution model for the species where you have data for and keep it for the extrapolation. 
 
The distribution models you can choose in PK-Sim consider different compound physicochemical properties that interact with species specific physiological properties (molecular weight, charge / pKa value, binding to plasma / tissue components, tissue composition / pH etc. ). Further, each model follows a slightly different hypothesis how those factors are combined resulting in observed distribution (volume of distribution).  The considered compound properties and method how there are used for estimating the distribution should be conserved across species. Otherwise, I would see a "break" in the underlaying hypothesis. 
 
Check out the [documentation for the different distribution models](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-compounds-definition-and-work-flow#adme-properties) implemented in PK-Sim and the linked primary literature to get a better feeling which model to choose.

Sorry for the long post.

Best
Henrik 

In short:
Pick one distribution model that allows you to describe the experimental data you have and stick with it for human PK prediction. If this does not work you might have missed a relevant process.
